### PR TITLE
(feat): Sending files

### PR DESCRIPTION
**Pull request by klinh5**
- Implement UI and functions

### CODE REVIEW

File: ClearKeep/Modules/Room/CKRoomViewController.swift [hunk 1]
```diff
@@ -2629,15 +2634,65 @@ extension CKRoomViewController {
...
     }
     
+    private func openDocumentWindow() {
+        documentPickerPresenter.delegate = self
+        documentPickerPresenter.presentDocumentPicker(with: [MXKUTI.data], from: self, animated: true) {
+        }
+    }
 }
 
-//extension UITextView {
```
Comment (sinbadflyce) [inline ClearKeep/Modules/Room/CKRoomViewController.swift]: Why did you remove these lines? @klinh5

Comment (klinh5) [reply ClearKeep/Modules/Room/CKRoomViewController.swift]: Unused & commented-out codes.
